$OpenBSD$

Revert the file back to ESR45.1.1, all changes were TB-specific and
not relevant to OpenBSD
--- toolkit/xre/nsEmbedFunctions.cpp.orig	Wed Jun 15 20:57:42 2016
+++ toolkit/xre/nsEmbedFunctions.cpp	Wed Jun 15 20:59:04 2016
@@ -619,8 +619,6 @@ XRE_InitChildProcess(int aArgc,
       mozilla::sandboxing::InitLoggingIfRequired();
 #endif
 
-      OverrideDefaultLocaleIfNeeded();
-
       // Run the UI event loop on the main thread.
       uiMessageLoop.MessageLoop::Run();
 
